web_src/js/features/repo-issue.js

```diff
@@ -680,3 +680,14 @@ export function initIssueTemplateCommentEditors($commentForm) {
     initCombo($(el));
   }
 }
+
+export function initArchivedLabelHandler() {
+  const archivedLabelEl = document.querySelector('.archived-label-hint');
+  if (!archivedLabelEl) {
+    return;
+  }
+  const archivedLabels = document.querySelectorAll('[data-is-archived]');
+  for (const label of archivedLabels) {
+    toggleElem(label, label.classList.contains('checked'));
+  }
+}
```

Comment:
This is an another case on where there is no checkbox but we are showing `archived` label if  that is selected and hence if that anchor tag contains `checked` class other wise we will hide. 
![image](https://github.com/go-gitea/gitea/assets/80308335/2a5535d6-21e3-456a-bf17-81e7719d45bd)